rest-api/tools/import_organizations.py

```diff
@@ -140,15 +141,27 @@ def _cleanup_old_entities(self, session, row_list, dry_run):
       for hpo in hpos_to_remove:
         old_hpo = self.hpo_dao.get_by_name(hpo)
         if old_hpo:
-          log_prefix = '(dry run) ' if dry_run else ''
-          logging.info(log_prefix + 'Deleting old HPO no longer in Google sheet: %s', old_hpo.name)
           hpo_id_list.append(old_hpo.hpoId)
           self.deletion_count += 1
 
       if hpo_id_list and not dry_run:
+        for hpo in hpo_id_list:
+          old_hpo = self.hpo_dao.get(hpo)
```

Comment:
I don't think we need to do this lookup again; we just looked them up in existing_hpos?

We do probably need logic above that won't bother to try to delete hpos again if they are already marked obsolete... that will get rid of the log spam of saying "we're going to delete X" over and over.